src/DGtal/images/doc/moduleImages.dox

```diff
@@ -394,7 +398,68 @@ The values associated to writing the points are adapted
 
 The use is the same that for ConstImageAdapter.
 
- \section dgtalImagesFunctions Useful classes and functions
+\subsection arrayimageadat ArrayImageAdapter (C++11 only)
```

Comment:
remove the C+11 only (not needed anymore).
and add yourself to the author list;)